src/app_service/service/stickers/service.nim

```diff
@@ -359,7 +359,7 @@ QtObject:
       preferredUsername: string) =
     let response = status_chat.sendChatMessage(
         chatId,
-        "Update to latest version to see a nice sticker here!",
+        "",
```

Comment:
Please put an emoji instead. Something like 🎨 or 🖼. If you use the second one, change the Image one with  📷.
There is no Sitcker emoji.

That text is what shows up in the Notification, so we need to show something for now

Reply:
What about :label:? (`label`)

Reply:
Not a fan

Reply:
OK :)

Reply:
if i put something here the problem is that we get additional space on the top of the sticker because the text message loader thinks there is a message so it's loading the component and then Elina reports a bug of sticker misalignment :P.  see:
<img width="633" alt="tx" src="https://user-images.githubusercontent.com/31625338/212121355-042f210f-9f83-44c4-9906-5db024f93a96.png">

The red empty boxes are this message "Update to latest version to see a nice sticker here!" which is hidden at the end

Reply:
I like that we're not loading the text message unnecessarily! 👍

Reply:
Also, since we're in the early stages of the app not sure there's value in replacing the text with something else for older clients.

Reply:
@jrainville since no text will appear at the end, do we really need to set even an emoji?

Reply:
The text is shown in the OS notification

Reply:
ok, I propose 2 things..either we leave as "Update to latest version to see a nice sticker here!" and we give it as `messageNotificationText`, separating it from the `messageText`, or we add the check in the UI to not show the StatusMessageText component if the messageText is equal to "Update to latest version to see a nice sticker here!", which has the risk of not displaying anything if there would be ever the case that someone types "Update to latest version to see a nice sticker here!" on purpose.

Reply:
None of them is good in my book 🙂 The text ends up being visible in the UI and not translatable. I'd prefer the emoji actually

Reply:
@caybro still, we have to do either. even with the emoji we'll get the StatusTextMessage rendered with no text so we'll be having the space on top (plus loading an additional component in the message which is not needed)

Reply:
and it's not visible in the UI, but in the OS notification

Reply:
That's still part of our application as we create that notification :)

Reply:
I don't get it, are you saying that we shall leave it as it is, just send one emoji instead and as a consequence load the StatusMessageText component in StatusMessage even though it has nothing to show?

Reply:
Hmm.. so we should probably fix the StatusTextMessage to not show the text if it contains this special single emoji placeholder

Reply:
We agreed with Alex to not send any text here and instead fix the notification side of the problem